spec/factories.rb

```diff
@@ -546,6 +546,16 @@
       end
     end
 
+    trait :with_cancelled_interview do
+      awaiting_provider_decision
+
+      after(:build) do |application_choice, _evaluator|
+        application_choice.status = :awaiting_provider_decision
```

Comment:
is this line needed? Isn't it covered by line 550 above?

Reply:
Line 550 will create a new application choice with that state. But if one is passed in to the factory, this line should ensure that its status is updated to `awaiting_provider_decision`